crates/telio-wg/src/wg.rs

```diff
@@ -292,7 +296,26 @@ impl WireGuard for DynamicWg {
         loop {
             if let Some(port) = self.get_interface().await?.listen_port {
                 if port > 0 {
-                    telio_log_trace!("Wireguard using port:{:?}", &port.to_string());
+                    telio_log_debug!("Wireguard using port:{:?}", &port.to_string());
+                    break Ok(port);
+                }
+            }
+            let elapsed_time = start.elapsed()?;
+            if elapsed_time > d {
+                break telio_err_with_log!(Error::PortAssignmentTimeoutError);
+            }
+
+            sleep(Duration::from_millis(10)).await;
+        }
+    }
+
+    async fn wait_for_proxy_listen_port(&self, d: Duration) -> Result<u16, Error> {
+        let start = std::time::SystemTime::now();
+        loop {
+            let ifa = self.get_interface().await?;
```

Comment:
What is ifa?  Maybe some smol comment on that? Just "Interface A", or maybe it has some smarter meaning?

Reply:
I think it is shorter variant of `iface`, which is shorter variant of `interface` :kekw:

Reply:
changed to interface, ifa is common shortening in kernel :sweat: